config/src/invariant.rs

```diff
@@ -31,3 +37,84 @@ impl Default for InvariantConfig {
         }
     }
 }
+
+impl InlineConfigParser for InvariantConfig {
+    fn config_key() -> String {
+        INLINE_CONFIG_INVARIANT_KEY.into()
+    }
+
+    fn try_merge(&self, configs: &[String]) -> Result<Option<Self>, InlineConfigParserError> {
+        let overrides: Vec<(String, String)> = Self::get_config_overrides(configs);
+
+        if overrides.is_empty() {
+            return Ok(None)
+        }
+
+        // self is Copy. We clone it with dereference.
+        let mut conf_clone = *self;
```

Comment:
ditto on prev comment

Reply:
Sure we can clone `self`, but I have to suppress clippy, that complains we should prefer `Copy` over `Clone`.

Reply:
Ahh gotcha, I'm good with it then if clippy complains otherwise!